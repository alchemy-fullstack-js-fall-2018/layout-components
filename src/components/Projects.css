p {
  font-size: 30px;
  text-align: center;
  min-height: 100%;
  font-family: "Helvetica Neue", Arial, sans-serif;
  position: relative;
  margin: 30px;
}
h3, h4 {
  text-align: center;
  font-size: 60px;
}
#projects {
  display: flex;
  justify-content: space-evenly;
}
