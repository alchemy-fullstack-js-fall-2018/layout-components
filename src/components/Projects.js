import Component from './Component';
import styles from './Projects.css';


export default class Projects extends Component {
  render() {
    const projectContent = `<h3>Projects</h3>

    <div id="projects"><h4>${this.props.project1}</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p>
    <h4>${this.props.project2}</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p>
    <h4>${this.props.project3}</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p></div>
    `;

    return `<section class="${styles.projectSection}">
      ${projectContent}
    </section>`;
  };
}
