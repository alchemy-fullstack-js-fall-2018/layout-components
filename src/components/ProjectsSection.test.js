import Projects from "./Projects";

describe("Projects component", () => {
  it("should be a project section", () => {
    const projects = new Projects({
      project1: "Dog match",
      project2: "Salmon Cookies",
      project3: "Day planner"
    });
    expect(projects.render()).toMatchSnapshot();
  });
});
