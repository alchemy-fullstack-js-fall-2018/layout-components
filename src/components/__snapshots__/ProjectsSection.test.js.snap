// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component should be a project section 1`] = `
"<section class=\\"projectSection\\">
      <h3>Projects</h3>

    <div id=\\"projects\\"><h4>Dog match</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p>
    <h4>Salmon Cookies</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p>
    <h4>Day planner</h4>
    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptatibus quasi consequuntur veniam, eaque numquam sint tenetur velit in, voluptas quisquam eos! Possimus adipisci natus obcaecati inventore quaerat facilis enim similique</p></div>
    
    </section>"
`;
